pkg/scaffold/addtoscheme.go

```diff
@@ -33,7 +33,7 @@ type AddToScheme struct {
 func (s *AddToScheme) GetInput() (input.Input, error) {
 	if s.Path == "" {
 		fileName := fmt.Sprintf("addtoscheme_%s_%s.go",
-			strings.ToLower(s.Resource.Group),
+			strings.ToLower(s.Resource.GoImportGroup),
```

Comment:
Nit: Can we use `s.Resource.GoImportGroup` without `strings.ToLower()`, since it has already been lowercased?